src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1291,6 +1281,29 @@ void PosixSignals::print_signal_handler(outputStream* st, int sig,
   st->cr();
 }
 
+void PosixSignals::print_signal_handlers(outputStream* st, char* buf, size_t buflen) {
+  st->print_cr("Signal Handlers:");
+  PosixSignals::print_signal_handler(st, SIGSEGV, buf, buflen);
```

Comment:
You shouldn't need the PosixSignals:: prefix in this method.

Reply:
Goo catch, fixed.

Reply:
Actually now that we are changing `PosixSignals::print_signal_handlers()` to `os::print_signal_handlers()` we do need the `PosixSignals::` prefix after all.